contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -545,7 +545,7 @@ def sync(self, pk, attributes=False, tags=False, files=False, assessment_items=F
                 attributes,
                 tags,
                 files,
-                tags,
+                assessment_items,
```

Comment:
As this fixes https://github.com/learningequality/studio/issues/3112 - this may be worth doing directly to hotfixes?

Could extract just the tests for this change and this change as a separate PR.

cc @bjester

Reply:
The issue is currently unprioritized, but if this is the only change that's needed, then that could be worth it. I'd rather do the full release flow unless it's a P0

Reply:
This was a regression caused by the release (by me), so might be worth doing.

Reply:
Oh gotcha, lets do it then @vkWeb

Reply:
@bjester yes sir - I'll do it.